include/rabbit_mqtt_frame.hrl

```diff
@@ -36,11 +36,12 @@
 %% connect return codes
 
 -define(CONNACK_ACCEPT,      0).
--define(CONNACK_PROTO_VER,   1). %% unacceptable protocol version
--define(CONNACK_INVALID_ID,  2). %% identifier rejected
--define(CONNACK_SERVER,      3). %% server unavailable
--define(CONNACK_CREDENTIALS, 4). %% bad user name or password
--define(CONNACK_AUTH,        5). %% not authorized
+-define(CONNACK_PROTO_VER,   1).   %% unacceptable protocol version
+-define(CONNACK_INVALID_ID,  2).   %% identifier rejected
+-define(CONNACK_SERVER,      3).   %% server unavailable
+-define(CONNACK_CREDENTIALS, 4).   %% bad user name or password
+-define(CONNACK_AUTH,        5).   %% not authorized
+-define(CONNACK_DUP_AUTH,    255). %% duplicate authentication attempt
```

Comment:
We can use a different value, I just picked the last one in the "reserved for future use" range:

http://docs.oasis-open.org/mqtt/mqtt/v3.1.1/csd02/mqtt-v3.1.1-csd02.html#_Toc385349257

Reply:
I think that would trip up some clients. I'm afraid if we are being forgiving in this case we should use a `CONNACK_ACCEPT` 🤷‍♂️

Reply:
I tried that first. If we do that, new channels and what-not are opened by the processor. I'll see if that can be skipped.

Reply:
Yeah, that won't be a problem.